### DISCUSSION TITLE
Custom directive: modifiers not parsed correctly when used with value

### DISCUSSION THREAD

**Discussion by leoboyerbx** [alpinejs/alpine#3586] [7. Bugs (archived)]
Hi, I found a bug with the extending system of Alpine.

When writing a custom directive and using both modifiers and value, like this: `x-foo.modifier1.modifier2:value="expression"`, the last modifier is not parsed correctly, it includes the ':value'. I believe this is not the expected behavior.

This stackblitz reproduces the issue: [https://stackblitz.com/edit/web-platform-rzrl7r?file=index.html](https://stackblitz.com/edit/web-platform-rzrl7r?file=index.html).

**Comment by ekwoka** [comment]
The value needs to be before the modifiers.

Just like with all of the built in and core plugins.
